R/read_file.R

```diff
@@ -39,13 +39,13 @@ read_file <- function(path) {
     "fst" = fst::read_fst(path),
     "rda" = read_rda(path),
     "rds" = readr::read_rds(path),
-    rlang::abort("Extension is supported but something unexpected happened.")
+    abort("Extension is supported but something unexpected happened.")
```

Comment:
changes in this file also seem unrelated to this PR

Reply:
https://github.com/2DegreesInvesting/PACTA_analysis/pull/178